cls/SourceControl/Git/WebUIDriver.cls

```diff
@@ -344,6 +344,41 @@ ClassMethod HandleRequest(pagePath As %String, InternalName As %String = "", Out
                     }
                     set handled = 1
                 }
+            } elseif (pathStart = "git-command-unstage") {
+                set flags = $get(%request.Data("flags",1),"")
+                if (flags '= "") {
+                    set flaglist = $listfromstring(flags,",")
+                    for i=1:1:$listlength(flaglist) {
+                        set flag = $listget(flaglist,i)
+                        set args($increment(args)) = flag
+                    }
+                }
+                set message = $get(%request.Data("message",1),"")
+                if (message '= "") {
+                    set args($increment(args)) = "-m"
+                    set args($increment(args)) = message
+                }
+                set details = $get(%request.Data("details", 1),"")
+                if (details '= "") {
+                    set args($increment(args)) = "-m"
+                    set args($increment(args)) = details
+                }
+                set command = $get(%request.Data("command",1),"")
+
+                set files = ##class(%Library.DynamicObject).%FromJSON($get(%request.Data("files",1),""))
```

Comment:
Between message, details, and files we're likely to trade <MAX ZF STRING> for webserver limitations on URL length with a truly large set of files to commit.

Let's move this to a POST body.